### ISSUE TITLE
Additions

### ISSUE THREAD

**Issue by justalever** [fpsvogel/learn-ruby#4]
Hey @fpsvogel!

Thanks for putting this together 👏 

I'm just throwing my hat in the ring to be linked here if that's of interest! No big deal if I don't make the cut.

I have a few Ruby/Rails-related links:

- [webcrunch](https://webcrunch.com) - A blog about modern web development with Rails/Ruby/Hotwire and Tailwind CSS
- [Rails UI](https://railsui.com) - A new UI component and template library for Rails apps.
- [Hello Rails](https://hellorails.io) - A course for beginner Rails developers or aspiring ones.
- [webcrunch on YouTube](https://youtube.com/webcrunch) - Primarily ruby/rails content with a mix of other web development tutorials and videos.

**Comment by fpsvogel** [comment]
Hey! I'm glad you suggested these, because I hadn't checked your YouTube channel in a while and wasn't aware of your recent videos. They look great.

I already had a link to your channel in the ["let's build" screencasts section](https://github.com/fpsvogel/learn-ruby#lets-build-screencasts) of my list, but now I've added your channel to the [topical screencasts section](https://github.com/fpsvogel/learn-ruby#topical-screencasts) as well. I also took the opportunity to remove some of the less active channels from the list, to reduce clutter.

Regarding blogs, there are so many of them out there that I limit the list to my favorites that I read often. Similarly for courses and UI libraries, there are a lot of them out them out there, so I tend to favor free resources (especially for beginners) or particular favorites.

I know a lot of these choices are subjective, and I hope my response doesn't come across as negative. I do feel the screencasts sections are better organized thanks to your reaching out. With your work better represented there, I hope anyone scrolling through the list can find you more easily and branch out to your other resources if they wish.
